sdk/identity/identity/src/credentials/managedIdentityCredential.ts

```diff
@@ -238,7 +255,17 @@ export class ManagedIdentityCredential implements TokenCredential {
 
     try {
       // Detect which type of environment we are running in
-      if (process.env.MSI_ENDPOINT) {
+      if (process.env.IDENTITY_ENDPOINT && process.env.IDENTITY_HEADER) {
+        // Running in App Service 2019-08-01
+        authRequestOptions = this.createAppServiceMsiAuthRequest(resource, clientId, "2019-08-01");
+        expiresInParser = (requestBody: any) => {
+          // Parses a string representation of the seconds since epoch into a number value
+          return Number(requestBody.expires_on);
```

Comment:
If I'm understanding this correctly, the server will now answer with a seconds since epoch string, like `"1560999478"`.